src/components/shared/TextAreaField/index.tsx

```diff
@@ -35,6 +39,7 @@ const TextAreaField = ({
         value={value}
         maxLength={maxLength}
       />
+      <HelpText>{`${2000 - value.length} characters left`}</HelpText>
```

Comment:
This should be moved out into `RequestDetails`. The count should be the responsibility of the form and not the text area. The max length wont always be 2000 characters.